data/human/deep missions.txt

```diff
@@ -2471,29 +2471,63 @@ mission "Stone of our Fathers 1"
 		conversation
 			`Helheim's spaceport has all the constant hustle and bustle of a boomtown. It's dirty, lively, and packed with workers. As you walk through the spaceport, an aged man in a worn and weathered roustabout jacket waves his arm toward you and approaches you.`
 			`	"You've got the look of one who knows a ship inside and out," he says. "Name's Skaldgar, ain't much for ships; earth, soil, and rock are my domain. But I do got a need to travel." He pauses as a wistful look briefly passes over his face. "I left, you see, decades ago. No goodbye. No note. Just... left. Living with others, taking care of others, hard on a man like me. I like solitude, peace, and work. But now my bones are brittle: the radiation, earth-rot as we call it, has taken hold and my time is running out. And I find, I don't want to be alone anymore."`
-			`	He stops and frowns. "Look at me, rambling like an ol' man. Captain, I'd like to pay you <payment> to take me home, to <destination>."`
+			`	He stops and frowns. "Look at me, rambling like an ol' man. Ser, Captain, I'd like to pay you <payment> to take me home, to <destination>."`
+			label choice
 			choice
 				`	"If you have coin, I have room."`
 				`	"I guess I can take a small detour."`
+				`	"Why are you calling me 'ser'?"`
+					goto ser
+					to display
+						not "Stone of our Fathers: ser"
+				`	"Does 'ser' just apply to men?"`
+					goto "ser 2"
+					to display
+						has "Stone of our Fathers: ser"
+						not "Stone of our Fathers: ser 2"
 				`	(Shake my head. I have better things to do than bus old men around.)`
 					goto decline
+
+			action
+				clear "Stone of our Fathers: ser"
+				clear "Stone of our Fathers: ser 2"
 			`	He nods, grunts, and heads to your ship with a small knapsack cradled under his arm, clearly ready to travel to <planet> at a moment's notice.`
 				accept
+
+			label ser
+			action
+				set "Stone of our Fathers: ser"
+			`	"You're not from around here, are ya? Ser's just what decent folk call each other in the Deep. Sign of respect and the like. Though I am hearing it less and less these days. Old ways being forgotten, but ain't that the way of things?"`
+			`	He pauses for a moment as if reflecting on this, then says, "Well, what do you say? <payment> for transportation to <planet>?"`
+				goto choice
+
+			label "ser 2"
+			action
+				set "Stone of our Fathers: ser 2"
+			`	The old man narrows his eyes at you and his mouth falls slightly open like he's trying to make sense of a puzzle. Finally he says, "What's the bits between a fella's legs gotta do with anything? I already said, 'Ser' is just what decent folk call each other; ain't nothing more to it. Don't matter if you're a guy, gal, or anything else. Hel, I reckon I'd even call an elf 'Ser' if they done me a good turn.`
```

Comment:
I'm sorry, but that just does not sound like Skaldgar.

Reply:
It's just a suggestion that would need to be polished to sound like Skaldar.

Reply:
I get that, but making it sound like Skaldgar is how I arrived at what is written above. He's an old miner grandpa, he's going to be colorful. Using gender or sex just didn't sound right, too textbook or too careful. That's where we arrive at the bits tween a fella's legs.

Reply:
_Ser_ was confused for _Sir_ and that was considered offensive enough to be removed. " the bits between a fella's legs" seems like it might be considered equally as offensive or taboo to some people and is inappropriate for the kids that play this game. What doesn't make sense is, Skaldgar is aware enough to use a gender neutral term but then quickly devolves into an overly colorful, rude, "old miner grandpa".  
Yes, my version might be too textbook, too sterile, and too careful but the goal is to let the player know about the use of _ser_ without offending anyone.

Reply:
Ser was not removed because it was offensive. It was removed because a few people were unaware of its meaning. 

I am a bit flabbergasted that you think "bits between legs" could be inappropriate for children playing this game. Like... do you think children don't know about their genitalia? Like... one day when they are old enough they find out about it - but before then they just assume there is nothing there? I am a father of two, and I can tell you there's never been a mystery for my kids about that. 

This is a game where people are being killed, genocide is happening, slavery is rampant, genetic experimentation, etc etc... but all of that is fine for a young mind - but you draw the line at acknowledging genitalia? _Dude._ 

To your next point, Skaldgar is not "aware enough to use a gender neutral term", he's not cutting edge. It's an old term that was just always gender neutral. 

And to your final point; you realize this is circular reasoning? I point out your suggestion isn't in his voice, you say you understand and recommend it be put in his voice, I point you that is exactly what I did, and now you say it doesn't matter it isn't in his voice? 

No one was offended in the past, just some people did not understand what it means. That is what we're addressing here.

Reply:
I personally don't find any of it offensive. I only want to point out what might be an issue to other people. In some cultures, just referencing genitalia is taboo. As, for the kids part, I agree with you that if they are playing this game, "bits between legs" shouldn't be an issue but not all people feel the same way. Why risk it just to add flavor, when flavor isn't really the goal of this whole new dialogue. 
Ser was removed because "some people did not understand what it means" is accurate but , at the time, the idea was put forth that we could not use it, because it was not gender neutral. To me, that means the fear was it could offend some people.   

I'm not trying to pick a fight I'm just pointing out areas that might become an issue later. I like to play devils advocate just to make people think. The best question to ask before you do anything is, "What if?". As soon as you think you thought of everything, some else comes along and finds the one thing you missed. 

I have made my alternative points of view clear enough. I'm fine with letting the other members help bene-dictator decide what to use. Use some of it, or use none of it, it's all fine by me.

Reply:
> Why risk it just to add flavor, when flavor isn't really the goal of this whole new dialogue.

Having a character sound like themselves is not "adding flavor", it's intrinsically good writing.  Maintaining their characterization is pivotal to making helping the audience believe they are interacting with a real person - i.e. a character.